test/cli/cache-keeps-print-options/cache-keeps-print-options.out

```diff
@@ -1,14 +1,16 @@
 class ::<root> < ::Object () @ (... removed core rbi locs ..., https://github.com/sorbet/sorbet/tree/master/bazel-out/host/genfiles/rbi/procs.rbi#L1)
   class ::CCCCCCCCC < ::Object () @ test/cli/cache-keeps-print-options/cache-keeps-print-options.rb:1
-  class ::<Class:CCCCCCCCC> < ::<Class:Object> () @ test/cli/cache-keeps-print-options/cache-keeps-print-options.rb:1
+  class ::<Class:CCCCCCCCC>[AttachedClass] < ::<Class:Object> () @ test/cli/cache-keeps-print-options/cache-keeps-print-options.rb:1
```

Comment:
How closely have you looked at all the snapshot changes? How carefully should I also be looking at them?

Reply:
Not super closely, though the ones that I inspected were all changes to include the `[AttachedClass]` type member. Maybe it would be a good idea to censor it?

Reply:
Sometimes what I do is I'll either manually or with a tool bucket the changes into different groups, which usually gets down to a handful of expectation changes (like maybe 10 individual changes) that I can't explain and thus look suspicious.

All grouped together like this, it's hard to tell whether there are any suspicious changes.

Reply:
That's a good idea. I'll try to bucket them into commits after grepping through the diff for common patterns.

Reply:
I went back through the exp file changes, and realized that we shouldn't be including AttachedClass in document-symbol.